cmd/hamctl/command/root.go

```diff
@@ -45,17 +56,10 @@ func NewRoot(version *string) (*cobra.Command, error) {
 				client.BaseURL = os.Getenv("HAMCTL_URL")
 			}
 
-			if client.Metadata.AuthToken == "" {
-				client.Metadata.AuthToken = os.Getenv("HAMCTL_AUTH_TOKEN")
-			}
-
 			var missingFlags []string
 			if service == "" {
 				missingFlags = append(missingFlags, "service")
 			}
-			if err := setCallerEmail(gitConfigAPI, &client, email); err != nil {
```

Comment:
Doesn't this make the gitConfigAPI unused and maybe even the whole local git config functionality could be removed? 🥳